test/test_test.go

```diff
@@ -81,6 +155,31 @@ func (d *draggable) DragEnd() {
 	d.wasDragged = true
 }
 
+type focusable struct {
+	widget.BaseWidget
+	focused bool
+}
+
+var _ fyne.Focusable = (*focusable)(nil)
```

Comment:
nit: these conformity assertions are usually above the type

Reply:
Did we agreed to that?

Reply:
You're right we did not explicitly agree to that, but in general vars before types